### PR TITLE
Relicense to Apache-2.0

### PR DESCRIPTION
**Pull request by kgugala**
Signed-off-by: Karol Gugala <kgugala@antmicro.com>

### CODE REVIEW

File: .github/workflows/install-and-test.yml [hunk 1]
```diff
@@ -612,13 +660,21 @@ jobs:
         #!/usr/bin/env python3
         # -*- coding: utf-8 -*-
         #
-        # Copyright (C) 2021  The SymbiFlow Authors.
-        #
-        # Use of this source code is governed by a ISC-style
-        # license that can be found in the LICENSE file or at
-        # https://opensource.org/licenses/ISC
-        #
-        # SPDX-License-Identifier: ISC
```
Comment (mithro) [inline .github/workflows/install-and-test.yml:621]: I think the indenting in this file is wrong?

Comment (kgugala) [reply .github/workflows/install-and-test.yml:621]: fixed

**Review by mithro — APPROVED** [review]
LGTM.

**Comment by mithro** [comment]
There seems to be a bunch of CI failures. I assume these are failing on master at the moment too?

**Comment by pgielda** [comment]
Last CI run was 10 months ago so its very probable. We probably should be running it weekly at least... But I doubt those fails are related to this PR...

**Pull request merged**